chrome/content/zotero/fileInterface.js

```diff
@@ -510,14 +510,17 @@ var Zotero_File_Interface = new function() {
 			format = "rtf";
 		}
 		
+		// determine locale preference
+		var locale = Zotero.Prefs.get('export.lastLocale');
```

Comment:
Take care of in https://github.com/rmzelle/zotero/commit/be6b5738fc1e787f36947514c5a7f6cb8950358d